### DISCUSSION TITLE
Special citation style

### DISCUSSION THREAD

**Discussion by Neradiabint** [Zettlr/Zettlr#4533] [Q&A]
Hello, 
As a newbie I hope this question is not too naive or posed to the wrong audience. 

I'm a PhD studend and just get started with markdown (and Pandoc etc.) and I so far have no idea of coding. So I want to check out if I can realize the citation style my supervisor requests, before I take a deep dive.   (I really would like to use Zettlr) 

These are the required specs for the citation: 
- generally footnote
- Every paper needs a full citation the first time it is quoted in a chapter (so not only once) 

Can someone help? I only need a hint if and how this is possible, I don't want to do all this useless citation work by hand. I use Zotero and already know in general how citaions work in zettlr, so it is more about the final editing that gives me headaches. 

My solution (but I have no idea if this works) would be to create several documents (one per chapter) to guarantee the complete citaion for first time references. But this would cause other problems, regarding an complete Index and the final bibliography. 

Thanks in advance, have a nice day!

**Comment by nathanlesage** [comment]
First, have a look at www.zotero.org/styles – this is a CSL style repository with numerous styles. I know that especially historians like to first cite fully and then abbreviated, so you may be able to find a style that does that already. The repository contains tons of styles, so the chances are good.

Second, if that doesn't work, you can create your own style. That could be a tad tricky, since you'll need to know XML and have to play around a little, but then it should be possibly to achieve what you seek.

**Comment by Neradiabint** [comment]
Thank you for your answer, but I'm afraid you missed the point, that the full citation is required with every new chapter. As far as I know CSL styles are not aware of chapters/headers.
